apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx

```diff
@@ -21,6 +22,7 @@ interface Props {
 
 const ModuleNavigation: FC<Props> = ({ nav, onItemClick, badge }) => {
   const [expand, setExpand] = useState(false)
+  const [hover, setHover] = useState(false)
```

Comment:
Not being used?

And setTimeout on l15?

Reply:
Ah must be some junk from testing - I'll cleanup !